build.gradle

```diff
@@ -216,7 +220,8 @@ processResources {
 processTestResources {
     //Don't waste time packaging our test data into the test jar
     exclude "**/large/**"
-    exclude "**/org/broadinstitute/hellbender/**"
+    //TODO: DON"T MERGE THIS!!!!!!!!11!!!11!!!
```

Comment:
Yeah - Turns out it was unnecessary.  I misunderstood how the exclusions were working, but it's all fixed.  I'll remove the commented out code.